_clanky/výběr-katalogizačního-systému-pro-město-Brno.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Výběr katalogizačního systému pro město Brno
+detail: true
+ref: výběr-katalogizačního-systému-pro-město-Brno
+lang: cs
+author: robert_spál
+date: 2020-12-16T07:00:00 +01:00
+---
+Článek se zaměřuje na různé aspekty výběru datového katalogu v prostředí české samosprávy. Na příkladu města Brna jsou zde do hloubky srovnány hlavně dva katalogizační systémy - CKAN a ArcGIS HUB. Specifikace a popsané funcionality těchto dvou systémů mohou posloužit jako vodítko při výběru vhodného systému i dalším organizacím.
+
+<!--more-->
+
+## Výběr katalogizačního systému pro město Brno
+
+Řeší vaše město výběr vhodného katalogizačního systému pro otevřená data? Jste na vážkách, jaký systém vybrat? Má vaše město už vybráno a běží vše hladce nebo se potýkáte s překážkami a problémy spojenými s vybraným systémem? To, že proces výběru vhodného systému není nic jednoduchého, Vám chceme ukázat na příkladu města Brna. V tomto článku se dočtete, jak k výběru město přistoupilo a také to, že výběr jednoho systému nemusí být vždy výběrem konečným.
+
+Kolem roku 2015 se problematika otevřených dat začala v Česku dostávat do popředí, a to i na úrovni krajských měst.  Města, jako byla například Praha, začala svoje data postupně otevírat. Každé z nich se však potýkalo na této cestě ke skutečnému otevření dat s celou řadou problémů a překážek. Jednou z hlavních otázek, na kterou je nutné hned na začátku celého procesu najít odpověď, je **výběr vhodného katalogizačního systému**. Bez něj by se správa otevřených dat stala velmi neefektivním a časově náročným procesem. Nebylo by možné data třídit, ukládat, registrovat nebo vyplňovat jim metadata. I pro uživatele je katalog důležitý, neboť jim umožňuje jednoduše vyhledat jak požadované datové sady a informace k nim, tak i například podmínky jejich užití.
```

Comment:
Tyhle vytučněné části jsou přesně ukázkou zbytečného formátování [zakázaného metodikou](https://docs.google.com/document/d/1q1YWfszPIeZitnLMI91vm1LsKqBhl-R05f6VhYwlcKo/edit#heading=h.pqga8qoud8d1). Co tím chceš říct? Že stačí číst jen to tučné? Nebo že "výběr katalogizačního systému" je důležitý pojem? To je snad jasné už z názvu článku.

Reply:
opravené - tučný text som odstránil